### PR TITLE
Added Unittests

### PR DESCRIPTION
**Pull request by NeonDmitry**

### CODE REVIEW

**Comment by NeonDmitry** [comment]
@NeonDaniel please review this, I has no access to add reviewers

File: .github/workflows/unit_tests.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      - name: Set up python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements/requirements.txt
+          pip install -r requirements/test_requirements.txt
+      - name: Get Credential
```
Comment (NeonDaniel) [inline .github/workflows/unit_tests.yml]: This `Get Credential` step can be removed for this plugin

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+.idea/workspace.xml
+
+.idea/workspace.xml
+
+
+.idea/workspace.xml
+
+.idea/workspace.xml
+
```
Comment (NeonDaniel) [inline .gitignore]: These lines can be replaced by just `.idea/`

File: tests/test_detect.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import json
```
Comment (NeonDaniel) [inline tests/test_detect.py:28]: This added file needs a license notice (https://gist.github.com/NeonDaniel/1652cbc98717643db50583d80274bca3#file-header-bsd-3-license)

**Review by NeonDaniel — CHANGES_REQUESTED** [review]
Couple small changes noted

File: tests/test_detect.py (hunk 3 above)
Comment (NeonDmitry) [reply tests/test_detect.py:28]: @NeonDaniel  resolved this small changes as you say, sir!

**Review by NeonDaniel — APPROVED** [review]

**Pull request merged**